packages/labs/src/index.md

```diff
@@ -0,0 +1,34 @@
+---
+reference: labs
+---
+
+@# Labs
+
+<div class="pt-callout pt-intent-warning pt-icon-info-sign">
+    <h5>Under construction</h5>
+    The **[@blueprintjs/labs](https://www.npmjs.com/package/@blueprintjs/labs)** NPM package contains **unstable React components under active development by team members**. It is an incubator and staging area for components as we refine the API design; as such, this package will never reach 1.0.0, and every minor version should be considered breaking.
+</div>
+
+- [`Select`](#labs/select-component) for selecting items in a list.
+@page select-component
+
+- [`Suggest`](#labs/suggest) for selecting items in a list, from a text input.
+@page suggest
+
+- [`MultiSelect`](#labs/multi-select) for selecting multiple items in a list.
+@page multi-select
+
+- [`Omnibox`](#labs/omnibox) is a macOS spotlight-style typeahead component.
+@page omnibox
+
+- [`QueryList`](#labs/query-list) is a higher-order component that provides interactions between a query string and a list of items.
+@page query-list
+
+- [`TagInput`](#labs/tag-input) is an input component for [`Tag`](#core/components/tag)s.
+@page tag-input
+
+- [`Popover2`](#labs/popover2) is an improvement over [`Popover`](#core/components/popover), using [Popper.js](https://popper.js.org/).
+@page popover2
+
+- [`Tooltip2`](#labs/tooltip2) is like [`Tooltip`](#core/components/tooltip), but uses [`Popover2`](#labs/popover2).
+@page tooltip2
```

Comment:
let's move all the `@page` statements to one list at the bottom to avoid breaking the list into separate markdown blocks.